File: CTDK/119/119.cpp
#include <iostream>
using namespace std;

int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;

	float ahh = 0;
	float at = 2;
	int i = 2;
	while (i <= n)
	{
		ahh = (at * at + 2) / (2 * at);
		i++;
		at = ahh;
	}
	cout << "So hang thu " << n << " : " << ahh << endl;
	return 0;
}

File: CTDK_Python/119/_119.py
n = int(input("Nhap n: "))
if (n == 1):
    ahh = 2
else:
    at = 2
    i = 2

    while (i <= n):
        ahh = (at * at + 2) / (2 * at)
        i += 1
        at = ahh

print("Ket qua:", ahh)